wagtailvideos/models.py

```diff
@@ -236,9 +236,7 @@ def get_transcode(self, media_format):
         except Transcode.DoesNotExist:
             return self.do_transcode(media_format)
 
-    def video_tag(self, attrs):
-        kw_attrs = {}
-
+    def video_tag(self, attrs, kw_attrs={}):
```

Comment:
With the above comments on the video tag, this would change to become
```python
def video_tag(self, attrs=None):
    if attrs is None:
        attrs = {}
    else:
        attrs = attrs.copy()  # Do not mutate the dict passed in by the caller
    # ...
```
And then use `attrs` instead of `kw_attrs` through the rest of the function.